packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -145,12 +160,42 @@ export const Subscriptions = ({
     [navigateToUrl, SUPPORT_FORM_URL]
   );
 
+  useEffect(() => {
+    const fetchSubscriptionPreviews = async () => {
```

Comment:
We need to get the invoice previews for the prices of the subscriptions, so that we can show the correct price and tax. We can't use the subsequent invoices, since it gets the next invoice of the active subscription, which might include credits and unpaid amounts, as shown in the screenshot below.

![image](https://user-images.githubusercontent.com/10620585/207380351-ac140e59-ad88-4500-ac69-1c1f0ea0022f.png)